WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerView.xaml

```diff
@@ -45,60 +45,46 @@
           </StackPanel>
         </TreeDataTemplate>
         <TreeDataTemplate DataType="ViewModels:SendTabViewModel">
-          <Button BorderThickness="0" Background="Transparent" Command="{Binding DoItCommand}" HorizontalAlignment="Stretch" HorizontalContentAlignment="Left">
-            <StackPanel Orientation="Horizontal" Spacing="6">
-              <DrawingPresenter Width="16" Height="16" Drawing="{DynamicResource WalletExplorerView_Send}" />
-              <TextBlock Text="{Binding Title}" VerticalAlignment="Center" />
-            </StackPanel>
-          </Button>
+          <StackPanel Orientation="Horizontal" Spacing="6">
```

Comment:
What happened with the Button here? Please give me some description of the new workflow.

Reply:
It works by selection now, we don't need buttons anymore, the buttons were an old hack from Lisbon!

Wallet Explorer is a selection based model... SelectedItem changes and this triggers opening the document.

Reply:
ACK